demos/python_demos/text_spotting_demo/README.md

```diff
@@ -0,0 +1,132 @@
+# Text Spotting Python* Demo
+
+This demo shows how to run Text Spotting models using OpenVINO&trade;.
```

Comment:
This is somewhat tautological. It would be nice if you could explain what text spotting actually _is_.

Reply:
added one more sentence